isthmus/build.gradle.kts

```diff
@@ -94,6 +95,8 @@ dependencies {
   implementation("org.immutables:value-annotations:2.8.8")
   annotationProcessor("org.immutables:value:2.8.8")
   testImplementation("org.apache.calcite:calcite-plus:${CALCITE_VERSION}")
+  testImplementation("org.apache.arrow:arrow-dataset:${ARROW_VERSION}")
+  testImplementation("org.apache.arrow:arrow-memory-netty:${ARROW_VERSION}")
```

Comment:
This is a great integration test, but I think it would be better if we moved it to the Arrow repo instead of adding Arrow as a dependency to Isthmus. What do you think?

Reply:
Ok, let me delete that part.

Reply:
Just deleted